templates/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -83,13 +53,90 @@ local tpus = import "../../tpus.libsonnet";
     },
   },
   local functional = base.Functional {
-    command+: [
-      "--max-epoch=1",
-      "--log_steps=10",
-      "--train-subset=valid",
-      "--input_shapes",
-      "128x64",
-    ],
+    command: utils.scriptCommand(
+      |||
+        python3 \
```

Comment:
if this test doesn't use any param overrides and is just the 1 step of `python3 ...`, then I think you don't need to use utils.scriptCommand but @will-cromar can tell us if there is any downside to doing so anyway

Reply:
There's no real downside. I think this way is better for consistency. If these commands are the same, the only thing I would recommend if factoring out `python3 ...` into a local string and prepending it as needed to these commands. Do you need the `pip install ...` for the functional test?

If there are some flags that vary, you can also use a format string to insert that values that vary like we do in the RoBERTa test: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/pytorch/nightly/roberta-pre.libsonnet

Reply:
I don't need pip install for functional test. It is required for the conv. test. There are about 5 flags that varies between them, but the main delta is, I couldn't make both added commands and substitution at the same time.

Reply:
It looks awkward if I have full commands as substitutions, such as substitute `"pip install --editable fairseq"` to one test and `""` to the other.